rosenpass/src/protocol.rs

```diff
@@ -211,6 +305,8 @@ pub struct Peer {
     pub session: Option<Session>,
     pub handshake: Option<InitiatorHandshake>,
     pub initiation_requested: bool,
+    pub cookie_tau: CookieSecret,
```

Comment:
I don't think storing the cookie in the peer is quite right, the cookie is bound to a specific handshake (through `sidi`), lets use it just for that…

Lets have it in the initiator handshake state…

Reply:
You're right, fixed